src/Badge.js

```diff
@@ -48,7 +48,7 @@ const Badge = styled.div`
   font-size: ${props => props.theme.fontSizes[0]}px;
   font-weight: 600;
   text-transform: uppercase;
-  letter-spacing: 0.05em;
+  letter-spacing: ${props => props.theme.letterSpacings.caps};
```

Comment:
The styled-system `theme` getter is generally better to use because it won't throw when `theme` or `theme.letterSpacings` is undefined. It can be used like this:

```
letter-spacing: ${theme('letterSpacings.caps')};`
```

See this https://github.com/jxnblk/styled-system/tree/ab0242c981fa1d94d29f607a42a9e9a284dd452c#theme

Reply:
Sweet, that sounds good, ill update.